editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/document_node_types.rs

```diff
@@ -110,6 +110,26 @@ static DOCUMENT_NODE_TYPES: &[DocumentNodeType] = &[
 		outputs: &[FrontendGraphDataType::Raster],
 		properties: node_properties::gpu_map_properties,
 	},
+	DocumentNodeType {
+		name: "Blur",
+		category: "Image Adjustments",
+		identifier: NodeIdentifier::new("graphene_core::raster::BlurNode", &[]),
+		inputs: &[
+			DocumentInputType::new("Image", TaggedValue::Image(Image::empty()), true),
+			DocumentInputType::new("Radius", TaggedValue::U32(3), false),
+			DocumentInputType::new("Sigma", TaggedValue::F64(1.), false),
+		],
+		outputs: &[FrontendGraphDataType::Raster],
+		properties: node_properties::blur_image_properties,
+	},
+	DocumentNodeType {
+		name: "Cache",
+		category: "Image Adjustments",
```

Comment:
This probably doesn't belong in this category either. Does it even need to be a user-accessible node?